### ISSUE TITLE
Secon display Start Menu.

### ISSUE THREAD

**Issue by EdShedisson** [valinet/ExplorerPatcher#3322]
So I'm running ExplorerPatcher 22621.3527.65.5 and Windows 11, OS Build 22631.3593.

When using DisplayFusion taskbar the Start menu and all of the tray pop-ups are showing up on the main monitor, even when clicked on the second one. But the Start menu shows up correctly when using default (non Win 10 Style) menu. Also there's no issue (obviously) when runnin' just stock Windows taskbar on the second screen. 

Can I somehow resolve the issue or bring tray icons on the second display without DisplayFusion taskbar?

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Is DisplayFusion the only other software you have installed that modifies the UI?

**Comment by EdShedisson** [comment]
> Please include what settings you changed in explorer patcher properties application. These should be screen shots. If you can, please change it to english first.
> 
> Is DisplayFusion the only other software you have installed that modifies the UI?

Here you go.
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/17c2eee6-1bc0-4a3f-b496-9104387e7108)
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/0761e23e-1cac-4e5f-8677-13262b7fe4c4)
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/d55c0e7c-0789-4294-bb00-4c947d6bcd69)
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/3fce391f-3ea0-482b-90e7-c7020054673f)
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/d209a88d-57f0-48b1-a59b-38569022a500)
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/44b2372b-51b3-4030-9738-adf825ad9638)
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/a2ca1809-06a0-4962-9287-0066527f9153)
Only DF and EP. I tried to resolve this issue with other mods, but with no luck so I did rollback to restore point.

**Comment by pyrates999** [comment]
I think this is already known:

duplicate of: https://github.com/valinet/ExplorerPatcher/discussions/2363

please close this.

**Comment by EdShedisson** [comment]
I don't think this is the case. My system tray icons are clearly visible and functioning as intended. Moreover, as I mentioned, I'm using the latest build of Windows 11 and Explorer Patcher. 
![image](https://github.com/valinet/ExplorerPatcher/assets/22825673/340a3fe7-3813-428d-a7c1-b817c88b153d)


> I think this is already known:
> 
> duplicate of: #2363
> 
> please close this.

**Comment by pyrates999** [comment]
you want the system tray icons to be shown on the second display?

**Comment by EdShedisson** [comment]
> you want the system tray icons to be shown on the second display?

Yeah. The thing is I'm running 3 monitor setup where one of them isn't close to my laptop and only serves as a "double" of my second monitor. And it's super inconvenient when I don't have any of tray controls over there. So the solution is DisplayFusion but EP doesn't want to work with it. At least not in the way the stock Windows 11 does.

**Comment by pyrates999** [comment]
Please wait for a developer to respond.
